app/assets/v2/js/pages/bounty_details.js

```diff
@@ -112,7 +112,11 @@ var heads = {
 var callbacks = {
   'github_url': link_ize,
   'value_in_token': function(key, val, result) {
-    return [ 'amount', token_value_to_display(val) + ' ' + result['token_name'] ];
+    var title = token_value_to_display(val) + ' ' + result['token_name'];
+    var title_expand = title + ' in funding from original funder.';
```

Comment:
@owocki I'm updating this to const just to be on the safer side + to avoid hoisting